inc/template-tags.php

```diff
@@ -301,9 +301,9 @@ function loose_gallery_content() {
 		$replacement = '';
 
 		$newcontent = preg_replace( $pattern, $replacement, $content, 1 );
-		$newcontent = apply_filters( 'the_content', $newcontent ); // WPCS: prefix ok.
+		$newcontent = apply_filters( 'the_content', $newcontent ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
 		$newcontent = str_replace( ']]>', ']]&gt;', $newcontent );
-		echo $newcontent; // WPCS: XSS OK.
+		echo esc_html( $newcontent ); // WPCS: XSS OK.
```

Comment:
@DivyanshKhanna26 here is supposed to be the content form wp editor... so it almost always contains HTML.